pkg/agent/manager/sync.go

```diff
@@ -225,3 +230,28 @@ func (m *manager) newCSR(spiffeID string) (pk *ecdsa.PrivateKey, csr []byte, err
 	}
 	return
 }
+
+func (m *manager) isNodeType(er *entryRequest) bool {
```

Comment:
small nit: may want to name this `isAgentType` instead, since the identity in question is meant for the agent... it just happens to be that in most cases, there is one agent per node, but maybe in the future that won't always be the case

Reply:
I renamed it as isAgentAlias...does this makes sense?

Reply:
👍 SGTM